From 07a54f53a1fbfa2381fceb5f3b9f1271af1ff552 Mon Sep 17 00:00:00 2001
From: "F. von Gellhorn" <flinux@vongellhorn.ch>
Date: Thu, 10 Jun 2021 17:22:10 +0200
Subject: [PATCH 1/1] Use pam security directory

- Pam currently uses /lib/security for it's files, so kwallet needs to put it there.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1a761c9..b8c91a3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -83,7 +83,7 @@ target_link_libraries (${library_name}
 file(GLOB_RECURSE ALL_CLANG_FORMAT_SOURCE_FILES *.cpp *.h)
 kde_clang_format(${ALL_CLANG_FORMAT_SOURCE_FILES})
 
-install(TARGETS ${library_name} DESTINATION ${KDE_INSTALL_LIBDIR}/security)
+install(TARGETS ${library_name} DESTINATION lib/security)
 
 configure_file(pam_kwallet_init.desktop.cmake ${CMAKE_BINARY_DIR}/pam_kwallet_init.desktop)
 
-- 
2.30.2

